### PR TITLE
Andgate

### PR DESCRIPTION
**Pull request by ajain35**
### Describe your change:

* [x] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### Checklist:
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [x] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [ ] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

File: boolean_algebra/andgate.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    """
+
+
+def and_gate(input_1: int, input_2: int) -> int:
+
+    return input_1 * input_2
+
+
+def main() -> None:
+    print("Truth Table of AND Gate:")
```
Comment (realstealthninja) [inline boolean_algebra/andgate.py:12]: please use a multi line string

File: boolean_algebra/andgate.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
+""" An AND Gate is a logic gate in boolean algebra which results to false(0)
+    if any of the input is 0, and True(1) if  both the inputs are 1.
+    """
+
+
+def and_gate(input_1: int, input_2: int) -> int:
```
Comment (realstealthninja) [inline boolean_algebra/andgate.py:6]: What about cases in which user provides a value that is not one or zero?

**Review by realstealthninja — CHANGES_REQUESTED** [review]

**Comment by CaedenPH** [comment]
@cclauss 
This pull request has already been implemented here 

https://github.com/TheAlgorithms/Python/blob/master/boolean_algebra/and_gate.py

This pr should be closed

**Comment by cclauss** [comment]
Duplicate https://github.com/TheAlgorithms/Python/blob/master/boolean_algebra/and_gate.py

**Pull request closed without merging**